test/e2e/network/netpol/model.go

```diff
@@ -181,6 +181,7 @@ type Pod struct {
 	Namespace  string
 	Name       string
 	Containers []*Container
+	ServiceIP  string
```

Comment:
This is the part that I'm finding difficult to understand, so each pod has its own service?

Reply:
If it is that way, then I finally got it.
Pods where assigned a service by name convention, creation happened after, but now we need the IP, so convention is not longer enough to probe them

Reply:
Yuppp